src/network/networkd-link.c

```diff
@@ -444,13 +442,8 @@ void link_enter_failed(Link *link) {
         }
 
         log_link_info(link, "Trying to reconfigure the interface.");
-        r = link_reconfigure(link, /* force = */ true);
-        if (r < 0) {
-                log_link_warning_errno(link, r, "Failed to reconfigure interface: %m");
-                goto stop;
-        }
-
-        return;
+        if (link_reconfigure(link, /* force = */ true) > 0)
```

Comment:
`>= 0` ?

Reply:
`link_reconfigure()` returns 1 when reconfiguration started. So, `> 0` is intentional.